python/paddle/fluid/tests/unittests/test_matmul_v2_op.py

```diff
@@ -1,64 +1,56 @@
-#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
-#
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+# 
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
-#
+# 
 #     http://www.apache.org/licenses/LICENSE-2.0
-#
+# 
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
 from __future__ import print_function
-
 import unittest
 import numpy as np
 from op_test import OpTest
 import paddle.fluid.core as core
-
 import paddle
 import paddle.fluid as fluid
 import paddle.fluid.framework as framework
+from paddle.fluid.framework import _test_eager_guard
 
 
 def reference_matmul(X, Y, transpose_X=False, transpose_Y=False):
-    """Reference forward implementation using np.matmul."""
-    # np.matmul does not support the transpose flags, so we manually
-    # transpose X and Y appropriately.
+    'Reference forward implementation using np.matmul.'
     if transpose_X:
-        if X.ndim == 1:
+        if (X.ndim == 1):
             X = X.reshape((X.size, ))
-        elif X.ndim == 2:
+        elif (X.ndim == 2):
             X = X.T
         else:
             dim = [i for i in range(len(X.shape))]
-            dim[-1], dim[len(X.shape) - 2] = dim[len(X.shape) - 2], dim[-1]
+            (dim[(-1)], dim[(len(X.shape) - 2)]) = (dim[(len(X.shape) - 2)],
+                                                    dim[(-1)])
             X = np.transpose(X, tuple(dim))
     if transpose_Y:
-        if Y.ndim == 1:
+        if (Y.ndim == 1):
             Y = Y.reshape((Y.size, ))
         else:
             dim = [i for i in range(len(Y.shape))]
-            dim[-1], dim[len(Y.shape) - 2] = dim[len(Y.shape) - 2], dim[-1]
+            (dim[(-1)], dim[(len(Y.shape) - 2)]) = (dim[(len(Y.shape) - 2)],
+                                                    dim[(-1)])
             Y = np.transpose(Y, tuple(dim))
-
     Out = np.matmul(X, Y)
-    if not Out.shape:
-        # We do not support 0-dimensional Tensors (scalars). So where
-        # np.matmul outputs a scalar, we must convert to a Tensor of
-        # shape (1, ) instead.
-        # Everywhere else, we are compatible with np.matmul.
-        Out = np.array([Out], dtype="float64")
+    if (not Out.shape):
+        Out = np.array([Out], dtype='float64')
     return Out
 
 
 class TestMatMulV2Op(OpTest):
-    """
-    case 1
-    """
+    '\n    case 1\n    '
```

Comment:
why change this?

Reply:
Hmmm, it's a change from astunparse, let's sync offline.